pyxem/signals/polar_diffraction2d.py

```diff
@@ -140,18 +128,22 @@ def get_pearson_correlation(self, mask=None, krange=None, inplace=False, **kwarg
             elements masked, False for elements unmasked
         krange: tuple
             The range of k values in corresponding unit for segment correlation (None if use
-            the entire pattern)
+            the entire pattern). Not compatible with ``inplace=True``.
         inplace: bool
             From hyperspy.signal.map(). inplace=True means the signal is
             overwritten.
 
         Returns
         -------
-        correlation: Signal2D
+        correlation: Signal2D,
+            The pearson rotational correlation when inplace is False, otherwise
+            return None
         """
         if krange is None:
             correlation = self.map(_pearson_correlation, mask=mask, inplace=inplace, **kwargs)
         else:
+            if inplace:
+                raise ValueError("`krange` is not compatible with `inplace=True`")
```

Comment:
Is this part necessary?  I don't really think that should be the case

Reply:
Actually, I was wrong, `map` is working fine and there is no issue, see #828!